test/testFans.py

```diff
@@ -56,8 +56,28 @@ def test_fans_initiation(self):
     def test_empty_body(self):
         result = get_cpp_function_list_with_fans_extension("a.cpp",
                                                            "int fun(a){})")
-        body = self.ext.method_open(result[0])
-        self.assertEqual([], body)
+        body = self.ext.open_method(result[0].filename)
+        self.assertEqual(None, body)
+
+    def test_open_function_infile(self):
```

Comment:
Please avoid reading and writing files in test if possible. What about mock the "open" method?